cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSICKANSink.java

```diff
@@ -349,6 +346,7 @@ private void persistAggregation(NGSIGenericAggregator aggregator, String service
                 aggregation += "," + jsonObject;
             }
         }
+        //entityId=aggregator.getEntityId();
```

Comment:
remove this line

Reply:
fixed in b179bd50cd3a0408217c4d5d0878910cc8530ce1